### ISSUE TITLE
[Android] MauiSplashScreen BaseSize doesn't work

### ISSUE THREAD

**Issue by janseris** [dotnet/maui#7304]
### Description

https://docs.microsoft.com/en-us/dotnet/maui/user-interface/images/splashscreen

> The base size of the splash screen can be specified by setting the BaseSize attribute to values that are divisible by 8:
> If you don't specify a BaseSize for a bitmap-based splash screen, the image isn't resized.

From this, I assumed that _if you specify a BaseSize for a bitmap-based image, the image is resized._

However, BaseSize does nothing on Android for me.
The image does not even fit screen, it overflows the screen.

![image](https://user-images.githubusercontent.com/64279914/169125204-98cd1bef-eccc-498c-a2a5-944fe6c18ef8.png)

Image:
![bird](https://user-images.githubusercontent.com/64279914/169126289-8b20f3a9-96b3-4be9-a97a-c8ee7ae12728.png)

Code: 

```
<MauiSplashScreen Include="Resources\bird.png" Color="Green" BaseSize="128,128"/>
```
or 128,160 to preserve original image aspect ratio

### Steps to Reproduce

Specify `BaseSize` attribute on `MauiSplashScreen` tag in `.csproj` when using a non-vector image.

### Version with bug

Release Candidate 3 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

**Comment by kristinx0211** [comment]
verified repro on android 11. repro project:
[MauiApp29.zip](https://github.com/dotnet/maui/files/8725799/MauiApp29.zip)

**Comment by mattleibow** [comment]
This is fixed in #8020 and BaseSize now controls the maximum size the image can be.
